contracts/mock-subcall/src/contract.rs

```diff
@@ -3,6 +3,11 @@ use soroban_sdk::{contract, contracterror, contractimpl, panic_with_error, Addre
 
 use crate::storage;
 
+mod subcall_wasm {
+    soroban_sdk::contractimport!(
+        file = "../../target/wasm32-unknown-unknown/release/mock_subcall.wasm"
+    );
+}
```

Comment:
this will make it impossible to build from scratch. Can we just directly invoke the contract without the use of a client?